packages/types/src/cart/common.ts

```diff
@@ -400,6 +400,10 @@ export interface CartLineItemDTO extends CartLineItemTotalsDTO {
    * When the line item was updated.
    */
   updated_at?: Date
+  /**
+   * When the line item was deleted.
+   */
+  deleted_at?: Date
```

Comment:
nit: nullable and rm `?` but the DTO's will need some rework later so you don't have to do it here